doc/sphinx-guides/source/admin/troubleshooting.rst

```diff
@@ -3,11 +3,35 @@
 Troubleshooting
 ===============
 
-This new (as of v.4.6) section of the Admin guide is for tips on how to diagnose and fix system problems. 
+Sometimes Dataverse users get into trouble. Sometimes Dataverse itself gets into trouble. If something has gone wrong, this section is for you.
 
 .. contents:: Contents:
 	:local:
 
+Using Dataverse APIs to Troubleshoot and Fix Problems
+-----------------------------------------------------
+
+See the :doc:`/api/intro` section of the API Guide for a high level overview of Dataverse APIs. Below are listed problems that support teams might encounter that can be handled via API (sometimes only via API).
+
+A Dataset Is Locked And Cannot Be Edited or Published
```

Comment:
It might be helpful to point out that one should be certain Dataverse is finished with a dataset (worst case: stop glassfish) before removing the lock.